### DISCUSSION TITLE
Headers?

### DISCUSSION THREAD

**Discussion by TecRecNecLec** [kokorin/Jaffree#323] [Q&A]
Hi all, im trying to set headers in ffmpeg execution.
is posible?, I don't see any method and addArgument/s don't work.

thanks!

**Comment by kokorin** [comment]
Hello, what arguments do you want to add exactly? Show ffmpeg/Jaffree log.

**Comment by TecRecNecLec** [comment]
Thanks for answer.

this is a example of what command I want generate.
ffmpeg -headers "Accept: text/xml" -I "http://streamurl" ... etc file.mp4

**Comment by kokorin** [comment]
Common, I'm not an oracle to help when you provide so limited information.
Please provide code and logs
